blackbox/docs/config/node.rst

```diff
@@ -680,6 +680,11 @@ Enterprise License
   | *Default:*  ``true``
   | *Runtime:*  ``no``
 
-  Setting this to ``false`` disables the `Enterprise Edition`_ of CrateDB.
+  This is a deprecated setting that allows you to disable the `Enterprise
+  Edition`_ features of CrateDB, enabling you to use all community edition
+  features of CrateDB without restrictions or license.
+  The setting will be removed in CrateDB 4.0 and instead you'll have to
+  `Acquire a license`_ or switch to the :ref:`community-edition` distribution.
 
 .. _`Enterprise Edition`: https://crate.io/enterprise-edition/
+.. _`Acquire a license`: https://crate.io/license-update/
```

Comment:
I think it is better to link to this site:
https://crate.io/pricing/

The license-update page is mainly for existing customers with an expired license.